_posts/2019-11-25-expert-review.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: Revising our program with the help of Open Science Leaders
+authors:
+- malvikasharan
+- yochannah
+- bebatut
+image: /images/2019-11-25-header-expert.jpeg
+photos:
+  name: isaacmsmith
+  url: https://unsplash.com/photos/Ij7h-m4nJ18
+---
+
+The [Open Life Science](https://openlifesci.org/) program is offered to the researchers and potential academic leaders who want to become ambassadors for Life Science in their communities. Developed as a part of the [Mozilla Open Leaders X](https://foundation.mozilla.org/en/opportunity/mozilla-open-leaders/open-leaders-x/), this 15-week long personal mentorship and cohort-based training enable sharing knowledge, connecting communities and empowering individuals involved in this program.
```

Comment:
Do we need this 1st paragraph as we publish the post on our website?

Reply:
I think it's useful in case people come to the blog post directly, and people who don't need to read it will skip that paragraph.